test/query_test/query_test.go

```diff
@@ -0,0 +1,128 @@
+package query_test
```

Comment:
The code doesn't change how query is built from the request URL, so this test is not needed.

Reply:
it does not change it but there was no such test anyway i think and since we are now starting to use timestamps it would be nice to verify the query is build properly for timestamps